Test/test2/Rlimitouts0.bpl.expect

```diff
@@ -0,0 +1,21 @@
+Prover error: line 105 column 10: max. resource limit exceeded
```

Comment:
This error message is very confusing. Line 105, column 10? What file does that refer to? It can't refer to the boogie program because that has 87 lines.